build/push

```diff
@@ -42,7 +57,10 @@ function TagAndPushImage() {
   REPO="$1"
   TAG="$2"
 
-  IMAGE_URI="${REPO}:${TAG}";
+  #Add an option to specify a custom TAG_SUFFIX 
+  #via environment variable. Default is no tag.
+  #Example suffix could be "-debug" of "-dev"
+  IMAGE_URI="${REPO}:${TAG}${TAG_SUFFIX}";
```

Comment:
Should the `TAG_SUFFIX` be defined only here? Or do we need to do it in Makefile also, since the push script completely works based on arguments passed on to it.

Reply:
The push script auto determines the tag, so it makes sense to keep this local for now. It only takes image URL as input (DIMAGE)